### ISSUE TITLE
Firefox long sessions closed after 1 hour

### ISSUE THREAD

**Issue by itayk15** [aerokube/selenoid#682]
Tests that are in long sessions not finished in Firefox because the session closed and the VNC shows a black screen (In chrome the tests finished)
To reproduce this issue you can just start a new container with firefox and don't touch it for 1 hour

**Comment by vania-pooh** [comment]
@itayk15 Selenoid default timeout is `1 minute`, so any sessions idle more than one minute will be closed. However if you set `sessionTimeout` capability you can increase this time up to `1 hour` by default. Please check that you are using default timeouts first.

**Comment by itayk15** [comment]
@vania-pooh, Of course, I also tried with ` -max-timeout` and it looks like there is an issue with Firefox images because on chrome that's working great

**Comment by vania-pooh** [comment]
@itayk15 what's inside Selenoid logs then?

**Comment by itayk15** [comment]
You can see that the container running only for 1 hour 
`
2019/03/10 13:15:23 [-] [INIT] [Loading configuration files...]
2019/03/10 13:15:23 [-] [INIT] [Loaded configuration from /etc/selenoid/browsers.json]
2019/03/10 13:15:23 [-] [INIT] [Video Dir: /opt/selenoid/video]
2019/03/10 13:15:23 [-] [INIT] [Logs Dir: /opt/selenoid/logs]
2019/03/10 13:15:23 [-] [INIT] [Using Docker API version: 1.38]
2019/03/10 13:15:23 [-] [INIT] [Timezone: Local]
2019/03/10 13:15:23 [-] [INIT] [Listening on :4444]
2019/03/10 14:22:07 [-] [NEW_REQUEST] [test] [159.65.113.141]
2019/03/10 14:22:07 [-] [NEW_REQUEST_ACCEPTED] [test] [159.65.113.141]
2019/03/10 14:22:07 [0] [LOCATING_SERVICE] [firefox] [65.0]
2019/03/10 14:22:07 [0] [USING_DOCKER] [firefox] [65.0]
2019/03/10 14:22:07 [0] [CREATING_CONTAINER] [selenoid/vnc_firefox:65.0]
2019/03/10 14:22:07 [0] [BAD_TIMEZONE] [GMT+2]
2019/03/10 14:22:07 [0] [STARTING_CONTAINER] [selenoid/vnc_firefox:65.0] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5]
2019/03/10 14:22:07 [0] [CONTAINER_STARTED] [selenoid/vnc_firefox:65.0] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5] [0.27s]
2019/03/10 14:22:07 [0] [SERVICE_STARTED] [selenoid/vnc_firefox:65.0] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5] [0.10s]
2019/03/10 14:22:07 [0] [PROXY_TO] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5] [http://172.17.0.3:4444/wd/hub]
2019/03/10 14:22:07 [0] [SESSION_ATTEMPTED] [http://172.17.0.3:4444/wd/hub] [1]
2019/03/10 14:22:10 [0] [SESSION_CREATED] [30f49b6b-02ff-4f50-b6fc-90fd287102d9] [1] [2.96s]
2019/03/10 14:24:44 [7] [CONTAINER_LOGS] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5]
2019/03/10 14:24:44 [8] [VNC_ENABLED] [30f49b6b-02ff-4f50-b6fc-90fd287102d9]
2019/03/10 14:24:51 [8] [VNC_CLIENT_DISCONNECTED] [30f49b6b-02ff-4f50-b6fc-90fd287102d9]
2019/03/10 14:24:51 [8] [VNC_SESSION_CLOSED] [30f49b6b-02ff-4f50-b6fc-90fd287102d9]
2019/03/10 14:24:56 [-] [NEW_REQUEST] [test] [159.65.113.141]
2019/03/10 14:24:56 [-] [NEW_REQUEST_ACCEPTED] [test] [159.65.113.141]
2019/03/10 14:24:56 [9] [LOCATING_SERVICE] [firefox] [65.0]
2019/03/10 14:24:56 [9] [USING_DOCKER] [firefox] [65.0]
2019/03/10 14:24:56 [9] [CREATING_CONTAINER] [selenoid/vnc_firefox:65.0]
2019/03/10 14:24:56 [9] [BAD_TIMEZONE] [GMT+2]
2019/03/10 14:24:57 [9] [STARTING_CONTAINER] [selenoid/vnc_firefox:65.0] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b]
2019/03/10 14:24:57 [9] [CONTAINER_STARTED] [selenoid/vnc_firefox:65.0] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b] [0.30s]
2019/03/10 14:24:57 [9] [SERVICE_STARTED] [selenoid/vnc_firefox:65.0] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b] [0.11s]
2019/03/10 14:24:57 [9] [PROXY_TO] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b] [http://172.17.0.4:4444/wd/hub]
2019/03/10 14:24:57 [9] [SESSION_ATTEMPTED] [http://172.17.0.4:4444/wd/hub] [1]
2019/03/10 14:24:59 [9] [SESSION_CREATED] [9be3829f-d363-4487-b3da-ae33a73be21d] [1] [2.99s]
2019/03/10 14:25:14 [15] [CONTAINER_LOGS] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b]
2019/03/10 14:25:14 [16] [VNC_ENABLED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:21 [16] [VNC_CLIENT_DISCONNECTED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:21 [16] [VNC_SESSION_CLOSED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:24 [18] [CONTAINER_LOGS] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b]
2019/03/10 14:25:24 [19] [VNC_ENABLED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:27 [15] [CONTAINER_LOGS_DISCONNECTED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:27 [19] [VNC_CLIENT_DISCONNECTED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:27 [19] [VNC_SESSION_CLOSED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:38 [18] [CONTAINER_LOGS_DISCONNECTED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:44 [20] [SESSION_DELETED] [30f49b6b-02ff-4f50-b6fc-90fd287102d9]
2019/03/10 14:25:44 [7] [CONTAINER_LOGS_DISCONNECTED] [30f49b6b-02ff-4f50-b6fc-90fd287102d9]
2019/03/10 14:25:44 [0] [REMOVING_CONTAINER] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5]
2019/03/10 14:25:44 [21] [SESSION_DELETED] [9be3829f-d363-4487-b3da-ae33a73be21d]
2019/03/10 14:25:45 [0] [CONTAINER_REMOVED] [0633060e3b343c2186495b21e249918acb416a4c88dd6af1bb223a85af139ba5]
2019/03/10 14:25:45 [9] [REMOVING_CONTAINER] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b]
2019/03/10 14:25:45 [9] [CONTAINER_REMOVED] [3412b3c768de892ab0315890b46426f22551675b6256d1275455592f32511f3b]
2019/03/10 14:25:51 [-] [NEW_REQUEST] [test] [159.65.113.141]
2019/03/10 14:25:51 [-] [NEW_REQUEST_ACCEPTED] [test] [159.65.113.141]
2019/03/10 14:25:51 [22] [LOCATING_SERVICE] [firefox] [65.0]
2019/03/10 14:25:51 [22] [USING_DOCKER] [firefox] [65.0]
2019/03/10 14:25:51 [22] [CREATING_CONTAINER] [selenoid/vnc_firefox:65.0]
2019/03/10 14:25:51 [22] [BAD_TIMEZONE] [GMT+2]
2019/03/10 14:25:51 [22] [STARTING_CONTAINER] [selenoid/vnc_firefox:65.0] [28f932ef357c329815bb2e2233a1c9d3bdd9ae9e60abdb0d8feb67cccd5f206d]
2019/03/10 14:25:51 [22] [CONTAINER_STARTED] [selenoid/vnc_firefox:65.0] [28f932ef357c329815bb2e2233a1c9d3bdd9ae9e60abdb0d8feb67cccd5f206d] [0.26s]
2019/03/10 14:25:51 [22] [SERVICE_STARTED] [selenoid/vnc_firefox:65.0] [28f932ef357c329815bb2e2233a1c9d3bdd9ae9e60abdb0d8feb67cccd5f206d] [0.10s]
2019/03/10 14:25:51 [22] [PROXY_TO] [28f932ef357c329815bb2e2233a1c9d3bdd9ae9e60abdb0d8feb67cccd5f206d] [http://172.17.0.3:4444/wd/hub]
2019/03/10 14:25:51 [22] [SESSION_ATTEMPTED] [http://172.17.0.3:4444/wd/hub] [1]
2019/03/10 14:25:53 [22] [SESSION_CREATED] [55440e8c-0dbc-464e-aeea-6b29f44a0c7a] [1] [2.96s]
`

**Comment by vania-pooh** [comment]
@itayk15 yep, in Firefox we are using Selenoid inside images too and there `-max-timeout` is still 1 hour.
